src/components/HelpModal/BorrowInterestHelpModal/index.tsx

```diff
@@ -20,6 +20,16 @@ export default function BorrowInterestHelpModal({
   const intl = useIntl();
   const { currentTheme } = useThemeContext();
 
+  return (
+    <div className="BorrowInterestHelpModal">
+      {text}
+      <style jsx>{`
+        .BorrowInterestHelpModal {
+          font-size: 12px;
+        }
+      `}</style>
+    </div>
+  );
   return (
```

Comment:
Should we comment out the below section?

Reply:
Need to delete it.